site-src/v1alpha2/guides/tcp.md

```diff
@@ -14,17 +9,47 @@ distribute the traffic with the following rules:
 - All TCP streams on port 8090 of the Gateway are forwarded to port 6000 of
   `my-bar-service` Kubernetes Service.
 
-Please note the following:
+In this example two `TCP` listeners will be applied to the [Gateway][gateway]
+in order to route them to two separate backend `TCPRoutes`, note that the
+`protocol` set for the `listeners` on the `Gateway` is `TCP`:
+
+```
+{% include 'v1alpha2/basic-tcp.yaml' %}
+```
+
+As a note to the above example: while it is possible to use metadata to route
+traffic from the `Gateway` to the underlying `TCPRoutes` (see the
+`spec.rules[].matches[]extensionRef` field in the [spec][tcproute]) such as is
+common for [HTTPRoutes][httproute] the most common use case for making these
+routing decisions with `TCPRoutes` is by port.
```

Comment:
Please move the explanation of extension point to the bottom.
We want to explain the core API capability first and make them clearly understood. And then, as an addendum, we mention that extensibility is possible.

Reply:
changed to a note in the bottom in c7bb1d621e3b879666de53e5f4337515637b8132